instrumentation/install/instrumentation.conf

```diff
@@ -1,2 +1,3 @@
 # service_name=default.service
 java_agent_jar=/usr/lib/splunk-instrumentation/splunk-otel-javaagent.jar
+exporter_otlp_endpoint=http://localhost:4317
```

Comment:
Should we include `resource_attributes=deployment.environment=<some default value>` in the default conf file, commented or uncommented?

Reply:
I was thinking this value would come from the installation script and be appended to this file at install time. Does that sound good? cc: @aunshc

Reply:
Makes sense to me.  If that's the case, should we have a default value for the installer script, maybe something like `prod`?  Or would it be a required argument?

Reply:
hmm not sure. @aunshc do you have an opinion on this?

Reply:
I do believe we should stay consistent with the current Java APM wizard in the first iteration; where the environment field is optional and we should only pass it to the installer script if it is set by the customer.